src/screen/dashboard.rs

```diff
@@ -302,6 +302,24 @@ impl Dashboard {
                                     self.split_pane(pane_grid::Axis::Horizontal, config)
                                 }
                                 command_bar::Buffer::Close => self.close_pane(),
+                                command_bar::Buffer::Replace(buffer) => {
+                                    let mut commands = vec![];
+
+                                    if let Some(pane) = self.focus {
+                                        if let Some(state) = self.panes.get_mut(&pane) {
+                                            state.buffer = Buffer::from(buffer);
+                                            self.last_changed = Some(Instant::now());
+                                            self.focus = None;
```

Comment:
Nit: you can use `self.focus.take` so you don't need to set it to `None` after.

Reply:
Done